File: api/examples/vb.net/BUBBLE_UPDATE.vb
<Serializable()>
Public Class BUBBLE_UPDATE
    Public BubbleID As Long = 0
    Public Options As Integer = 0
    Public DNSID As Long = 0
    Public ProxyList As New List(Of Long)
    Public VPNID As Long = 0
    Public Launch As New LAUNCH_ACTIONS
    Public BorderColor As String = String.Empty
    Public SpoofSettings As New Dictionary(Of Integer, String)
End Class

File: api/examples/c#/BUBBLE_UPDATE.cs
using System;
using System.Collections.Generic;

[Serializable()]
public partial class BUBBLE_UPDATE
{
    public long BubbleID = 0L;
    public int Options = 0;
    public long DNSID = 0L;
    public List<long> ProxyList = new List<long>();
    public long VPNID = 0L;
    public LAUNCH_ACTIONS Launch = new LAUNCH_ACTIONS();
    public string BorderColor = string.Empty;
    public Dictionary<int, string> SpoofSettings = new Dictionary<int, string>();
}